src/main/java/org/gridsuite/shortcircuit/server/service/ShortCircuitWorkerService.java

```diff
@@ -101,17 +101,6 @@ protected void sendResultMessage(AbstractResultContext<ShortCircuitRunContext> r
                 resultContext.getRunContext().getUserId(), additionalHeaders);
     }
 
-    @Override
-    protected void publishFail(AbstractResultContext<ShortCircuitRunContext> resultContext, String message) {
-        ShortCircuitRunContext context = resultContext.getRunContext();
-        String busId = context.getBusId();
-        Map<String, Object> additionalHeaders = new HashMap<>();
-        additionalHeaders.put(HEADER_BUS_ID, busId);
```

Comment:
are we sure we can remove this?

Reply:
Yes because it is set in the run message already, so if it is defined in the run message it will be forwarded in the DLQ